### ISSUE TITLE
ReferenceError: firstToUpper is not defined

### ISSUE THREAD

**Issue by abernix** [msavin/Mongol#97]
Hey,

`msavin:mongol                       1.1.0`

**This only happens on the pro-version** (I tried again with the same project non-pro and it works fine):

```
Exception in template helper: ReferenceError: firstToUpper is not defined
    at Object.Mongol.Collection (http://localhost:3200/packages/msavin_mongol.js?fa41ba1a0ec246cf6f7d836c39e686aaeafc51b1:192:97)
    at Object.Template.Mongol_trash.helpers.collectionCount (http://localhost:3200/packages/meteortoys_toypro.js?92de65bdef5693b87da1407413a601753f9f882a:458:57)
    at bindDataContext (http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2880:16)
    at Blaze._wrapCatchingExceptions (http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:1651:16)
    at http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2928:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:3476:12)
    at wrapHelper (http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2927:27)
    at Object.Spacebars.call (http://localhost:3200/packages/spacebars.js?7bafbe05ec09b6bbb6a3b276537e4995ab298a2f:172:18)
    at Template.Mongol_trash.Blaze._TemplateWith.Spacebars.include.HTML.DIV.Blaze.If.HTML.SPAN.class (http://localhost:3200/packages/meteortoys_toypro.js?92de65bdef5693b87da1407413a601753f9f882a:371:26)
    at null.<anonymous> (http://localhost:3200/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2626:44)
```

Looks like the method is just not defined at [this line in the source](https://github.com/msavin/Mongol/blob/6212e8609a7044010cfbc6f8e559870f57746c6a/lib/common.js#L150).  It's not defined anywhere in the package, and that's not a function in my browser. :)

**Comment by abernix** [comment]
I guess I should point out that everything still works, it just brings that up each time you load the Toys with Ctrl-M. :smile_cat:

**Comment by masiello-verys** [comment]
I'm on non pro. **Only have Mongol installed.**
I'm getting this same exception any time I open Mongol via Ctrl-M. It throws the exception three times with each open. It doesn't seem to affect the functionality of Mongol as I can still use it without other problems.

**Comment by saturnaut** [comment]
I'm on non pro. **Only have Mongol installed.**
I'm getting this same exception any time I open Mongol via Ctrl-M. It throws the exception three times with each open. It doesn't seem to affect the functionality of Mongol as I can still use it without other problems.

**Comment by abernix** [comment]
@saturnaut  Strange.  When I de-register my "pro" version, this doesn't happen.

Either way, as you stated, it outputs three times for me as well and it doesn't affect functionality, but it's mildly annoying as it gets in the way of my other debug code.

In addition to the above error, I also get the following error (before the three instances of the firstToUpper error) which seems to be happening in the same section of code:

```
Exception in template helper: TypeError: Cannot read property 'collections' of undefined
    at Object.Template.Mongol_header_pro.helpers.collection (http://localhost:3200/packages/meteortoys_toypro.js?92de65bdef5693b87da1407413a601753f9f882a:337:13)
```

As a dev hack to fix the `firstToUpper` problem originally mentioned, I just locally defined a firstToUpper() method:

``` javascript
function firstToUpper(s) { return s; }
```

**Comment by abernix** [comment]
I _should_ mention that this locally defined `firstToUpper` method clearly doesn't do what it's supposed to do, but the check that it's making isn't relevant to my system, so it's not necessary that it work. :smiley:

**Comment by saturnaut** [comment]
@abernix Good idea on that workaround. I do a ton of logging while working on Meteor apps and that error was really getting in the way.

Also, I'm not seeing the other error you mentioned (cannot read property 'collections' of undefined). I am only seeing one other error which is the font not loading (issue #98).

**Comment by msavin** [comment]
Corrected in 1.1.2. Going to close the issue, but please let me know if it persists.

**Comment by abernix** [comment]
Works.  Thanks for the fix @msavin. :)
